### ISSUE TITLE
Bootstrap - Header banner bugs / improvements

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1337]
I've been doing some accessibility testing and have found a few issues (accessibility & otherwise) with the BS banner heading functionality. Some of them are already listed in https://github.com/thexerteproject/xerteonlinetoolkits/issues/1129 but I will close that and list current issues here as there is a lot of discussion of things that are fixed in that issue.

- [ ] Scroll down hint & arrow doesn't do anything when clicked - the pointer cursor that you see on hover suggests that it should. Perhaps change so it auto-scrolls down on click.
- [ ] Scroll down hint & arrow doesn't reappear if you scroll down and then scroll back up.
- [ ] If a fixed heading height is set, it can be too small to show all of the heading text - this should be considered a minimum height  instead.
- [ ] Tabbing / keyboard only access becomes tricky with a full screen banner - you can get stuck in a loop and unable to get down to main content (see 1st page of release notes for an example)
- [ ] When collapse banner is off and you scroll down, the section menu first appears below the banner but then jumps up to overlap it as you scroll further down.
